lang/python/python-boto3/Makefile

```diff
@@ -0,0 +1,64 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=boto3
+PKG_VERSION:=1.9.135
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/b/boto3
+PKG_HASH:=5e5805992e873e5687b5ef3b4c56c386ccb4df1c3364f8b8601d289e2f275be1
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(BUILD_VARIANT)-boto3-$(PKG_VERSION)
+
+PKG_MAINTAINER:=Daniel Danzberger <daniel@dd-wrt.com>
+PKG_LICENSE:=Apache-2.0
```

Comment:
Can add `PKG_LICENSE_FILES:=LICENSE`.

Reply:
ok, I will drop the python2 package.
Shall I remove the PKG_LICENSE when adding the PKG_LICENSE_FILES:=LICENSE ?

Reply:
> Shall I remove the PKG_LICENSE 

No. PKG_LIECENSE needs to be there.

Reply:
done